.github/ISSUE_TEMPLATE/pre-release-operator-rust-deps.md

```diff
@@ -6,6 +6,11 @@ labels: ['epic']
 assignees: ''
 ---
 
+<!--
+    Make sure to update the link in '.github/ISSUE_TEMPLATE/release.md' when
+    you change the front matter above.
+-->
+
```

Comment:
I assume this can't sit above frontmatter (or at least it's probably hard to tell without creating a whole template)?

Reply:
I don't think it can sit above of the frontmatter, because at least the code highlighting broke locally. So I assumed that it wouldn't work on GitHub as well.